labs/lab10/deployment-script-lab10.sh

```diff
@@ -0,0 +1,51 @@
+#/bin/bash
+
+echo Parameters
+resourceGroupName="PubSubEvents"
+location="westeurope"
```

Comment:
Instead of hardcoding the location. I'd put pass it as an argument, with a flag such as `--location` and default value `west europe`, like this:
`location=${location:-westeu}